src/Juvix/Compiler/Backend/Markdown/Data/Types.hs

```diff
@@ -234,6 +234,11 @@ instance-- (MK.IsInline TextBlock) =>
           xs
       )
 
+nullMk :: Mk -> Bool
+nullMk = \case
+  MkNull -> True
+  _ -> False
```

Comment:
what about concat null null?

Reply:
I was using it internally after the mk is reduced, but we can it more general. np.

Reply:
Done